samples/zigbee/light_switch/README.rst

```diff
@@ -144,6 +144,11 @@ Button 2:
 
     Pressing this button for a longer period of time decreases the brightness of the **LED 4** of the connected light bulb.
 
+Button 4:
```

Comment:
Can we agree on a single button for all DK-based Zigbee samples?
I find it confusing, that bulb uses button1, while switch uses button4, the coordinator uses button2 and the shell uses button1 too.

Reply:
@tomchy Totally agree but these buttons were available on given samples - I guess previously assigned buttons were not unified across the samples. Reorganization would mean changing e.g. button for light_switch-related behavior etc. I think such action should be a separate task.

Reply:
I do not agree with that.

Current assignments:
- Button 1 & 2: application specific (On/Off/Dim/Permit devices)
- Button 3: Sleepy behavior (currently only ZEDs)
- Button 4: F&B (currently only the target side is supported)

Since the Button 3 is using long presses right now (you have to press it while booting) I propose to either:
 - Change the logic and use the RESET button (as discussed before)
 - Reuse the F&B button, as in the weather station
 
 @adsz-nordic @sebastiandraus @FilipZajdel Any thoughts on the proposal?

Reply:
+1 for Button 4, should be aligned with most of the sample. Maybe common code used for handling identify mode can be wrapped in a function in `zigbee_app_utils` library?

Reply:
@sebastiandraus Unfortunately Button4 is already assigned to light_bulb, shell and template as identify button. Button3 seems best candidate since it is not used except for SED trigger in light_switch. If we accept dual role in light_switch for Button3, it could be used.

Reply:
Regarding refactoring of the samples - shouldn't it be a separate task?

Reply:
That's my point - use the same button for identifying and factory reset across all or at least most Zigbee samples, so it's compatible also with weather station. Then the common code may be wrapped if it's feasible.

Reply:
@sebastiandraus  Thanks for explanation, same approach was proposed by @tomchy to combine identification and factory reset buttons. I will adjust the implementation to make it easier across multiple samples.
@tomchy even suggested to move factory reset functionality outside app utils and form another library but I would prefer to do it in a separate task. as it will need also additional documentation, etc. For now I'll stick with app utils.